src/editor/codeeditor.cpp

```diff
@@ -20,9 +20,9 @@ CodeEditor::CodeEditor(QWidget *parent) : QPlainTextEdit(parent)
 
     //read settings
     QSettings settings(QApplication::organizationName(), QApplication::applicationName());
-    QString analizerFontSize = settings.value("analizerFontSize").toString();
-    QString analizerFontName = settings.value("analizerFontName").toString();
-    QString analizerStyle = settings.value("analizerStyle").toString();
+    QString analizerFontSize = settings.value("editorFontSize").toString();
```

Comment:
analyzer...

Reply:
I changed key names that better represent it sence.
all new key names in storeconf.cpp